test/test_CellArray.jl

```diff
@@ -250,9 +260,9 @@ end
 					A[ix,iy] = A[ix,iy] * A[ix,iy];
 				    return
 				end
-				A.data.=3;                 @cuda blocks=size(A) matsquare2D_CUDA!(A); CUDA.synchronize(); @test all(Base.Array(A.data) .== 9)
-				J.data.=3; J_ref.data.=36; @cuda blocks=size(J) matsquare2D_CUDA!(J); CUDA.synchronize(); @test CUDA.@allowscalar all(J .== J_ref)
-				C.data.=2; G.data.=3;      @cuda blocks=size(C) add2D_CUDA!(C, G);    CUDA.synchronize(); @test all(Base.Array(C.data) .== 5)
+				A.data.=3;                 @cuda blocks=size(A) matsquare2D_CUDA!(A); synchronize(); @test all(Base.Array(A.data) .== 9)
```

Comment:
Did we not just change it the other way round? To be checked...

Reply:
Indeed. I fixed it to match `main`